### PR TITLE
Import the sampler

### PR DESCRIPTION
**Pull request by richfitz**
This is a direct import of the sampler. I've not done any real code tidyup yet

### CODE REVIEW

**Comment by richfitz** [comment]
I have no idea why travis status is not turning up here, the setup should the same as elsewhere.

Is it that case that we don't really run this returning the states/particles very often (that needs to be sped up if it is commonly done)?

File: R/parameters.R [hunk 1]
```diff
@@ -190,3 +190,164 @@ parameters <- function(
...
+  beta = 0.1,
+  seed_SSP = 10,
+  dt = 0.25){
+  N_age <- 17
+  
+  age_lim <- c(0, 5, 10, 15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80)
+  age_lim_polym <- c(0, 5, 10, 15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70)
+  
+  ## Severity parameters from spreadsheet
+  path <- sircovid_file("extdata/severity.csv")
```
Comment (MJomaba) [inline R/parameters.R]: I'm sure it's fine but I don't understand this line. do we have a function called sircovid_file() generated somewhere?

File: R/sampler.R [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+plot_particles <- function(particles, particle_dates, data, data_dates, ylab){
+  quantiles<-array(0,dim=c(nrow(particles),3))
+  for (i in 1:nrow(particles)){
+    quantiles[i,]<-quantile(particles[i,],c(0.025,0.5,0.975))
+  }
+  plot(particle_dates,particles[,1],type='l',col="#ff444477",ylab=ylab,ylim=c(min(particles),max(particles)))
+  for (i in 2:1000){
+    lines(particle_dates,particles[,i],col="#ff444477")
+  }
```
Comment (MJomaba) [inline R/sampler.R]: This would fail if there are less than 1000 particles I believe.

File: R/parameters.R (hunk 1 above)
Comment (richfitz) [reply R/parameters.R]: it's in util.R

File: R/sampler.R (hunk 2 above)
Comment (richfitz) [reply R/sampler.R]: thanks, I've generalised this now

File: R/parameters.R [hunk 3]
```diff
@@ -190,3 +190,164 @@ parameters <- function(
...
+  #Set up the heterogeneous offspring distribution #dnbinom(x=0,mu=2.2, size = 0.16)
+  #trans_classes <- 3
+  #trans_profile <- array(c(rep(.65,N_age),rep(.2,N_age),rep(.15,N_age)), c(N_age,trans_classes))
+  #trans_increase <- array(c(rep(0,N_age),rep(1,N_age),rep(10,N_age)), c(N_age,trans_classes))
+  
+  trans_classes <- 1
+  trans_profile <- array(c(rep(1,N_age)), c(N_age,trans_classes))
+  trans_increase <- array(c(rep(1,N_age)), c(N_age,trans_classes))
+  
+  #TODO: flexible seeding
```
[the remark this answers is not in the document]
Comment (richfitz) [reply R/parameters.R]: done thanks

**Pull request merged**